<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mkoq_kssrr</class>
 <widget class="QDialog" name="mkoq_kssrr">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>458</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>mkoq_kssrr</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="mkoq_kssrr_xxprg_l">
     <property name="text">
      <string>=mkoq_kssrr=</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mkoq_kssrr_xxsma_l">
     <property name="text">
      <string>=Мялик К.О.=</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="mkoq_kssrr_exit_c">
     <property name="text">
      <string>vixod</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="mkoq_kssrr_grid_c">
       <property name="text">
        <string>vibrat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mkoq_kssrr_back_c">
       <property name="text">
        <string>nazad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mkoq_kssrr_next_c">
       <property name="text">
        <string>vpered</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="mkoq_kssrr_xxtitle_l">
     <property name="text">
      <string>mkoq_kniga schetov (KS)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="mkoq_kssrr_b1">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="7" column="0">
       <widget class="QLabel" name="mkoq_kssrr_rubdb_l">
        <property name="text">
         <string>mko_summa DB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="mkoq_kssrr_rubdb_e">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="mkoq_kssrr_to_e">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mkoq_kssrr_to_l">
        <property name="text">
         <string>mko_operacia</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mkoq_kssrr_rub_l">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_data_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mkoq_kssrr_dokk_l">
        <property name="text">
         <string>mko_dokument</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>12</number>
        </property>
       </layout>
      </item>
      <item row="5" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="mkoq_kssrr_ks_l">
          <property name="text">
           <string>mko_Korr schet</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_kr_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_ksn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_s_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_sn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mkoq_kssrr_data_l">
        <property name="text">
         <string>mko_data operacii</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mkoq_kssrr_s_l">
        <property name="text">
         <string>mko_schet</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_dokk_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_dokn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mkoq_kssrr_dokd_l">
          <property name="text">
           <string>ot</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_dokd_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="7" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="mkoq_kssrr_rubkr_l">
          <property name="text">
           <string>mko_summa KR</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_kssrr_rubkr_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mkoq_kssrr_data_e</tabstop>
  <tabstop>mkoq_kssrr_dokk_e</tabstop>
  <tabstop>mkoq_kssrr_dokn_e</tabstop>
  <tabstop>mkoq_kssrr_dokd_e</tabstop>
  <tabstop>mkoq_kssrr_to_e</tabstop>
  <tabstop>mkoq_kssrr_s_e</tabstop>
  <tabstop>mkoq_kssrr_sn_e</tabstop>
  <tabstop>mkoq_kssrr_kr_e</tabstop>
  <tabstop>mkoq_kssrr_ksn_e</tabstop>
  <tabstop>mkoq_kssrr_rubdb_e</tabstop>
  <tabstop>mkoq_kssrr_rubkr_e</tabstop>
  <tabstop>mkoq_kssrr_grid_c</tabstop>
  <tabstop>mkoq_kssrr_back_c</tabstop>
  <tabstop>mkoq_kssrr_next_c</tabstop>
  <tabstop>mkoq_kssrr_exit_c</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
